.github/workflows/release.yml

```diff
@@ -76,4 +76,10 @@ jobs:
     - name: Run y.sh cargo build
       run: |
         EMBED_LTO_BITCODE=1 CHANNEL="release" ./y.sh cargo build --release --manifest-path tests/hello-world/Cargo.toml
-        # TODO: grep the asm output for "call my_func" and fail if it is found.
+        asm=$(objdump -dj .text tests/hello-world/target/release/hello_world)
+        if echo asm | grep -q "call .*mylib.*my_func" ; then 
+          echo "call my_func found in asm"
```

Comment:
Please add in this string something saying that it should not be found because LTO should have removed it and that this tests LTO.